packages/website/src/theme/MDXComponents/RuleAttributes.tsx

```diff
@@ -1,73 +1,113 @@
+import type { RuleMetaDataDocs } from '@site/../utils/dist/ts-eslint/Rule';
 import { useRulesMeta } from '@site/src/hooks/useRulesMeta';
-import type { TSESLint } from '@typescript-eslint/utils';
 import React from 'react';
 
+import type { FeatureProps } from './Feature';
+import { Feature } from './Feature';
 import styles from './RuleAttributes.module.css';
 
-export function RuleAttributes({ name }: { name: string }): JSX.Element | null {
+const getRecommendation = (docs: RuleMetaDataDocs): [string, string] => {
+  return docs.recommended === 'strict'
+    ? ['🔒', 'strict']
+    : docs.requiresTypeChecking
+    ? ['🧠', 'recommended-requiring-type-checking']
+    : ['✅', 'recommended'];
+};
+
+export function RuleAttributes({ name }: { name: string }): React.ReactNode {
   const rules = useRulesMeta();
   const rule = rules.find(rule => rule.name === name);
-  if (!rule) {
+  if (!rule?.docs) {
     return null;
   }
+
+  const features: FeatureProps[] = [];
+
+  if (rule.docs.recommended) {
+    const [emoji, recommendation] = getRecommendation(rule.docs);
+    features.push({
+      children: (
+        <>
+          Extending{' '}
+          <a
+            href={`http://localhost:3000/docs/linting/configs#${recommendation}`}
+            target="_blank"
+          >
+            <code className={styles.code}>
+              "plugin:@typescript-eslint/{recommendation}"
+            </code>
+          </a>{' '}
+          in an{' '}
+          <a
+            href="https://eslint.org/docs/latest/user-guide/configuring/configuration-files#extending-configuration-files"
+            target="_blank"
+          >
+            ESLint configuration
+          </a>{' '}
+          enables this rule.
+        </>
+      ),
+      emoji,
+    });
+  }
+
+  if (rule.fixable) {
+    features.push({
+      children: (
+        <>
+          Some problems reported by this rule are automatically fixable by the{' '}
+          <code>--fix</code>{' '}
+          <a
+            href="https://eslint.org/docs/latest/user-guide/command-line-interface#--fix"
+            target="_blank"
+          >
```

Comment:
question - did we want to put the `--fix` inside the link?

Reply:
Yeah I think that makes sense.